canonical-web-and-design/dqlite.io
# Fix webpack config includes

## Done

Fix require paths for rules in webpack config

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8037
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Run `./run exec yarn build-js` and check that there are no errors
